pkgs/top-level/python-packages.nix

```diff
@@ -15571,6 +15571,8 @@ self: super: with self; {
 
   roonapi = callPackage ../development/python-modules/roonapi { };
 
+  root = callPackage ../development/python-modules/root { };
```

Comment:
```suggestion
  root = callPackage ../development/python-modules/root {
    inherit (pkgs) root;
  };
```

Reply:
This is a commo  pattern in `python-packages.nix`. I'm surprised that it isn't documented.